app/javascript/components/widgets/twitter/tweet_body.jsx

```diff
@@ -10,6 +10,8 @@ import {
 } from '@twitter/tweet_content';
 import { colors, fonts, fontSizes } from '../../../lib/theme';
 
+const Entities = require('html-entities').AllHtmlEntities;
```

Comment:
Prefer es6 import style.

I don't think we want to allow `AllHtmlEntities` here (e.g. something like `<script>doSomethingEvil()</script>` would be bad here). What is this allowing and can we use an explicit whitelist?

Reply:
I'm not sure, this is the way it's recommended in https://www.npmjs.com/package/html-entities, and it doesn't seem to work any either way

Reply:
Nvm, just saw Gerald's stuff above. I'll look into that